src/components/chip/selectable-chip/index.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  StandaloneChevronDownRegularIcon,
+  StandaloneCircleXmarkRegularIcon,
+} from '@deriv/quill-icons'
+import { forwardRef } from 'react'
+import {
+  ChipIconSizes,
+  chipBaseClassnames,
+  ChipStandaloneIconSizes,
+  chipBaseVariant,
+  chipSelectionClassnames,
+} from '../chip.classnames'
+import { SelectableChipProps } from '../types'
+import qtMerge from 'qtMerge'
+
+export const SelectableChip = forwardRef<
+  HTMLButtonElement,
+  SelectableChipProps
+>(
+  (
+    {
+      className,
+      children,
+      dismissible = false,
+      icon: Icon,
+      isDropdownOpen,
+      dropdown = false,
+      labelTag,
+      onChipSelect,
+      onDismiss,
+      size,
+      ...rest
+    },
+    ref,
+  ) => {
+    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+      if (dismissible || dropdown) return
+      const target = event.currentTarget
+      const isSelected = target.getAttribute('data-state') === 'selected'
+      const selected_state = isSelected ? '' : 'selected'
+      target.setAttribute('data-state', selected_state)
+      onChipSelect?.(event, !isSelected)
+    }
+
+    const handleDismiss = (
+      event: React.MouseEvent<SVGSVGElement, MouseEvent>,
+    ) => {
+      onDismiss?.(event)
+    }
+
+    return (
+      <button
```

Comment:
Can we use `Button` component from Quill package?

Reply:
@likhith-deriv we can not use the button component here as this one is a separate component with its own styling. Including the Button component will require overwriting a lot of styles.